### PR TITLE
Update VSIX metadata

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: build/VSIX.targets [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
   <Target Name="_LocateMSBuildExe" Condition="Exists('$(MSBuildProgramFiles32)')">
     <ItemGroup>
-      <MSBuild15ExePaths Include="$(MSBuildProgramFiles32)\Microsoft Visual Studio\2017*\**\MSBuild\15.0\Bin\MSBuild.exe" />
+      <MSBuild15ExePaths Include="$(MSBuildProgramFiles32)\Microsoft Visual Studio\**\MSBuild\15.0\Bin\MSBuild.exe" />
```
Comment (pranavkm) [inline build/VSIX.targets]: My VS install is under Microsoft Visual Studio\Preview dir now.

Comment (joeloff) [inline build/VSIX.targets]: TeamCity 10.0.4 I know has support for VS 2017 to figure out all the locations for things like MS BUILD BTW.

**Comment by joeloff** [comment]
:shipit:

File: build/VSIX.targets (hunk 1 above)
Comment (pranavkm) [inline build/VSIX.targets]: I'd rather not split this off as a separate step though. Doing that requires using the msbuild runner

**Pull request merged**